bdk-ffi/src/descriptor.rs

```diff
@@ -266,6 +266,10 @@ impl Descriptor {
         let key_map = &self.key_map;
         descriptor.to_string_with_secret(key_map)
     }
+
+    pub(crate) fn is_multipath(&self) -> bool {
```

Comment:
Just out of curiosity, why are all of these access modifiers `crate` level? Clearly has no effect on the bindings but seems like a scope definition that doesn't actually apply

Reply:
good question/point-